sdk/include/opentelemetry/sdk/trace/samplers/probability.h

```diff
@@ -0,0 +1,52 @@
+#pragma once
```

Comment:
I think this file shouldn't be in this PR.

Reply:
Removed.